website/static/js/fangorn.js

```diff
@@ -1742,12 +1743,22 @@ var FGItemButtons = {
                         className: 'text-success'
                     }, 'Create Folder'));
                 if (item.data.path) {
-                    rowButtons.push(
-                        m.component(FGButton, {
-                            onclick: function(event) {_removeEvent.call(tb, event, [item]); },
-                            icon: 'fa fa-trash',
-                            className : 'text-danger'
-                        }, 'Delete Folder'));
+                    if (preprintPath && folderContainsPreprint(item, preprintPath)) {
+                        rowButtons.push(
+                            m.component(FGButton, {
+                                icon: 'fa fa-trash',
+                                tooltip: 'This folder contains a Preprint. You cannot delete Preprints, but you can upload a new version.',                                        
```

Comment:
This should be more vague. "This folder contains an un-deletable item such as a primary Preprint file or a checked-out file. You cannot delete blah blah"